### PR TITLE
adding rental html and css

### PR DESCRIPTION
**Pull request by delta-black5**

### CODE REVIEW

**Review by cpotter88 — APPROVED** [review]
Some spelling errors, but looks good.

File: assets/Dumpster Rental.html [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+              <div class="card-header py-3">
+                <h4 class="my-0 fw-normal">Medium Dumpster</h4>
+              </div>
+              <div class="card-body">
+                <h1 class="card-title pricing-card-title">$15<small class="text-body-secondary fw-light">/mo</small></h1>
+                <ul class="list-unstyled mt-3 mb-4">
+                  <li>Medium Dump?</li>
+                  <li>Trashly hurmor</li>
+                  <li>We are trashly people</li>
+                  <li>Call for qoute</li>
```
Comment (budtriplett) [inline assets/Dumpster Rental.html:144]: sp

File: assets/Dumpster Rental.html [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+          <div class="col">
+            <div class="card mb-4 rounded-3 shadow-sm">
+              <div class="card-header py-3">
+                <h4 class="my-0 fw-normal">Medium Dumpster</h4>
+              </div>
+              <div class="card-body">
+                <h1 class="card-title pricing-card-title">$15<small class="text-body-secondary fw-light">/mo</small></h1>
+                <ul class="list-unstyled mt-3 mb-4">
+                  <li>Medium Dump?</li>
+                  <li>Trashly hurmor</li>
```
Comment (budtriplett) [inline assets/Dumpster Rental.html:142]: sp

File: assets/Dumpster Rental.html [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+              <div class="card-header py-3 text-bg-primary border-primary">
+                <h4 class="my-0 fw-normal">Big Dumpster</h4>
+              </div>
+              <div class="card-body">
+                <h1 class="card-title pricing-card-title">$29<small class="text-body-secondary fw-light">/mo</small></h1>
+                <ul class="list-unstyled mt-3 mb-4">
+                  <li>Big Dump?</li>
+                  <li>Clean up your act</li>
+                  <li>We have big bin</li>
+                  <li>Call for qoute</li>
```
Comment (budtriplett) [inline assets/Dumpster Rental.html:161]: sp

File: assets/Dumpster Rental.html [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+                <td><svg class="bi" width="24" height="24"><use xlink:href="#check"></use></svg></td>
+              </tr>
+              <tr>
+                <th scope="row" class="text-start">5 Pickups</th>
+                <td></td>
+                <td><svg class="bi" width="24" height="24"><use xlink:href="#check"></use></svg></td>
+                <td><svg class="bi" width="24" height="24"><use xlink:href="#check"></use></svg></td>
+              </tr>
+              <tr>
+                <th scope="row" class="text-start">Umlimited Pickups</th>
```
Comment (budtriplett) [inline assets/Dumpster Rental.html:216]: sp

File: assets/Dumpster Rental.html [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+          <div class="col">
+            <div class="card mb-4 rounded-3 shadow-sm">
+              <div class="card-header py-3">
+                <h4 class="my-0 fw-normal">Small Dumpster</h4>
+              </div>
+              <div class="card-body">
+                <h1 class="card-title pricing-card-title">$0<small class="text-body-secondary fw-light">/mo</small></h1>
+                <ul class="list-unstyled mt-3 mb-4">
+                  <li>Small dump?</li>
+                  <li>Curb your enthusiam</li>
```
Comment (budtriplett) [inline assets/Dumpster Rental.html:125]: sp

**Review by budtriplett — APPROVED** [review]
Looks good. I added comments to lines I saw with spelling errors to help ensure none get mixed in your revisions.

**Pull request merged**